crates/astria-gossipnet/Cargo.toml

```diff
@@ -4,12 +4,13 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-libp2p = { version = "0.51", features = ["gossipsub", "mdns", "noise", "macros", "ping", "tcp", "tokio", "yamux"] }
+libp2p = { version = "0.51", features = ["kad", "gossipsub", "identify", "mdns", "noise", "macros", "ping", "tcp", "tokio", "yamux"] }
+multiaddr = "0.17"
 
 color-eyre = { workspace = true }
 futures = { workspace = true }
-tokio = { workspace = true, features = ["macros", "rt-multi-thread", "signal"] }
+tokio = { workspace = true, features = ["macros", "rt-multi-thread", "signal", "sync"] }
```

Comment:
Should this feature (actually, most of them) be under the normal dependencies? These look like they are only used as part of tests and live under `dev-dependencies`